### DISCUSSION TITLE
Rapsberry Pi 4 binary installation via kickstart?

### DISCUSSION THREAD

**Discussion by cakrit** [netdata/netdata#12261] [Q&A]
In the reddit post below, a user was asked Ng about the installation on a pi 4, as kickstart is complaining about openssl missing. Doesn't our binary installation work on Pis? 
https://www.reddit.com/r/netdata/comments/t2pxwm/netdata_on_raspberry_pi_4_running_home_assistant/

**Comment by Ferroin** [comment]
In theory it should, except:

* The user can still explicitly request a local build.
* We don’t _currently_ recognize Raspbian as a Debian derivative, so we don’t actually use native packages for it in the kickstart script.
* The CPU architecture reported by `uname -m` may not exactly match what we are looking for to try and use a static build. I’ve asked on Slack for anyone who has a Pi 4 to help a bit with this.

**Reply by cakrit** [reply]
@cpipilas we will buy one ASAP. Please speak with Tania so we can arrange procurement, I will let her know. Just discuss where we want to send it.

**Reply by ilyam8** [reply]
> The CPU architecture reported by uname -m may not exactly match what we are looking for to try and use a static build.

```cmd
pi@rpi4b-2:~ $ uname -m
armv7l
```

Yesterday we installed Netdata on 4 Pi 4B devices using kickstart script, it went without problems.

**Reply by Ferroin** [reply]
I’ve gotten internal confirmation from some people who actually have RPis that the last point is a non-issue.

This means that this particular user is almost certainly asking explicitly for a local build, and we should confer about this with them.

**Reply by cakrit** [reply]
Ok, I will ping the user again in Reddit to come here and give us more details.

**Reply by ilyam8** [reply]
@cakrit consider asking to open a bug report. It makes troubleshooting much easier for us.
